src/vp9hdec/decode_hybrid_vp9.cpp

```diff
@@ -2350,6 +2394,19 @@ VAStatus Intel_HybridVp9Decode_MdfHost_PadFrame (
     dwWidth         = pFrameResource->Frame.dwWidth;
     dwHeight        = pFrameResource->Frame.dwHeight;
 
+    if ((pFrameResource->pMdfDevice != pMdfDecodeEngine->pMdfDevice) ||
+        (pFrameResource->iMdfDeviceTsc != pMdfDecodeEngine->iMdfDeviceTsc))
+    {
+	// Decoder like libyami may call vaContextDestroy/Create to recreate media ctx and MdfDevice
```

Comment:
remove libyami reference from comment.  This code can be executed by any player that does a port reconfiguration due to new size of surfaces.